docs/blockchain/contracts/nft-contract/nft-tables.experimental.md

```diff
@@ -0,0 +1,812 @@
+| ------------------------- | -------- | ------------------------------------------------------------------------ |
+| min_price                 | asset    | Minimum offer price in UOS (Default: 1 UOS)                              |
+| min_duration              | uint32_t | Minimum duration for offer in seconds (Default: 86400 sec or 1 day)      |
+| max_duration              | uint32_t | Maximum duration for offer in seconds (Default: 15552000 sec or 180 day) |
+| max_active_offer_per_user | uint32_t | Maximum active offer per user (Default: 50)                              |
+
+Most relevant actions: `stofrcfg.a`
+
+-   `cleos` Query Example
+
+```sh
+cleos get table eosio.nft.ft eosio.nft.ft offercfg.a
+```
+
+-   `curl` query example
+
+```sh
+curl <NODEOS_API_IP>/v1/chain/get_table_rows -X POST -d '{"scope":"eosio.nft.ft", "code":"eosio.nft.ft", "table":"offercfg.a", "json": true}'
+```
+
+## next.nftofr
+
+-   Table: `next.nftofr`
+-   Code: `eosio.nft.ft`
+-   Scope: `0`
+-   Key: N/A
+
+The table stores information about the ID of the next created Uniq offer
+
+| Fields | Type     | Description                                      |
+| ------ | -------- | ------------------------------------------------ |
+| value  | uint64_t | ID that the next created Uniq offer will receive |
+
+Most relevant actions: `mknftofr.a`
+
+-   `cleos` Query Example
+
+```sh
+cleos get table eosio.nft.ft 0 next.nftofr
+```
+
+-   `curl` query example
+
+```sh
+curl <NODEOS_API_IP>/v1/chain/get_table_rows -X POST -d '{"scope":"0", "code":"eosio.nft.ft", "table":"next.nftofr", "json": true}'
+```
+
+## nftoffer.a
+
+-   Table: `nftoffer.a`
+-   Code: `eosio.nft.ft`
+-   Scope: `Uniq ID`
+-   Key: `id`
+
+The table stores information about the offer made by the buyer for specific Uniq ID
+
+| Fields               | Type                 | Description                                                                                  |
+| -------------------- | -------------------- | -------------------------------------------------------------------------------------------- |
+| offer_id             | uint64_t             | Uniq offer ID                                                                                |
+| buyer                | name                 | Buyer who makes the offer for the Uniq                                                       |
+| receiver             | std::optional\<name> | Users who will receive the Uniq if offer is accepted, if specified                           |
+| price                | asset                | Offer price in UOS                                                                           |
+| promoter_basis_point | uint16_t             | UOS share received by the promoter with this offer. Specified in basis points. 1 means 0.01% |
+| expiry_date          | time_point_sec       | Expiry date of the offer in seconds from the EPOCH                                           |
```

Comment:
Description should be
Expiry date
`time_point_sec` type is internally the number of seconds from EPOCH, but it is shown like "2024-02-13T04:25:25".

Reply:
updated